visidata/tests/test_commands.py

```diff
@@ -50,6 +50,7 @@ def isTestableCommand(longname, cmdlist):
                  'addcol-expr': 'Unit',          # just copy the column
                  'addcol-incr-step': '2',
                  'setcol-incr-step': '2',
+                 'setcol-format-enum': '1=cat',
```

Comment:
Each time we add a new `input` command, we need to add it to `test_commands.py`.

Reply:
Yes, and let's do `'1=cat 2=dog'` to test multiple, as long as we're here.